http-client-spi/src/main/java/software/amazon/awssdk/http/SdkHttpFullRequest.java

```diff
@@ -34,7 +34,7 @@
  * <p>All implementations of this interface MUST be immutable. Instead of implementing this interface, consider using
  * {@link #builder()} to create an instance.</p>
  */
-@SdkPublicApi
+@SdkProtectedApi
```

Comment:
This is kinda public since we expose it in the signers.

Reply:
good call